pages/app-manager/[jobId].jsx

```diff
@@ -1,178 +1,394 @@
-                    fillRule="evenodd"
-                    d="M7.21 14.77a.75.75 0 01.02-1.06L11.168 10 7.23 6.29a.75.75 0 111.04-1.08l4.5 4.25a.75.75 0 010 1.08l-4.5 4.25a.75.75 0 01-1.06-.02z"
-                    clipRule="evenodd"
-                  />
-                </svg>
-              </a>
-            </nav>
+            <Link
+              className="underline"
+              href={`/student/profile/${getValue().toString().replaceAll('-', '')}`}
+            >
+              view
+            </Link>
           </div>
-        </div>
-      </div>
-    </div>
+        ),
+      },
+      {
+        header: 'Date Applied',
+        id: 'lastUpdated',
+        accessorFn: (row) =>
+          new Intl.DateTimeFormat('en-US', {
+            month: 'short',
+            day: 'numeric',
+            year: 'numeric',
+          }).format(new Date(row.lastUpdated)), //Change to abbrev month spelled out, then day, and then year
+        sortingFn: 'datetime',
+      },
+    ],
+    []
   );
-};
+  const quarter_sort = (rowA, rowB, columnId) => {
+    const rowA_split = rowA.original.applicant.studentProfile.graduationDate.split(' ');
+    const rowB_split = rowB.original.applicant.studentProfile.graduationDate.split(' ');
 
-//Displaying table of applicants
-const ApplicantsCard = (props) => {
-  const { applicants } = props;
+    const year_sort = rowA_split[1] < rowB_split[1] ? -1 : rowA_split[1] == rowB_split[1] ? 0 : 1;
+    //If the years are different, sort by year
+    if (year_sort != 0) return year_sort;
+    //If the years are the same, sort by quarter
+    const quarter_priority = (A) => {
+      A = A.toLowerCase();
+      if (A == 'spring') return 0;
+      if (A == 'summer') return 1;
+      if (A == 'fall') return 2;
+      if (A == 'winter') return 3;
+    };
+    const rowA_quarter = quarter_priority(rowA_split[0]);
 
+    const rowB_quarter = quarter_priority(rowB_split[0]);
+    return rowA_quarter < rowB_quarter ? -1 : rowA_quarter == rowB_quarter ? 0 : 1;
+  };
+
+  const [{ pageIndex, pageSize }, setPagination] = useState({
+    pageIndex: 0,
+    pageSize: 5,
+  });
+
+  //TODO: Update all page sections to use cursor instead, also set pageSize to negative when requesting previous page and abs when setting next page
+  //TODO: Add prefetching next page, probably with prefetchQuery or fetchQuery
+
+  const router = useRouter();
+  const fetchDataOptions = {
+    router,
+    pageIndex,
+    pageSize,
+  };
+  const pagination = useMemo(
+    () => ({
+      pageIndex,
+      pageSize,
+    }),
+    [pageIndex, pageSize]
+  );
+
+  const applicantQuery = useQuery({
+    queryKey: ['applicants-count', router],
+    queryFn: () => fetchApplicantCount(router),
+    keepPreviousData: true,
+    staleTime: Infinity,
+  });
+
+  const [applicantCount, setApplicantCount] = useState(0);
+  const [pageCount, setPageCount] = useState(0);
+
+  const { status, data, error, isFetching, isPlaceholderData, isPreviousData } = useQuery({
```

Comment:
We probably have to store data in state to update data when a tag is changed. See https://tanstack.com/table/v8/docs/examples/react/editable-data